tools/shell_utils.sh

```diff
@@ -25,5 +25,5 @@ python_venv() {
   pip3 install -r "${SCRIPT_DIR}"/requirements.txt
 
   shift
-  python3 "${SCRIPT_DIR}/${PY_NAME}.py" "$*"
+  python3 "${SCRIPT_DIR}/${PY_NAME}.py" "${@}"
```

Comment:
this isnt strictly necessary for this pr as i switched the hook to using the python file directly

i still think this is a good idea tho, so included it.